### PR TITLE
fiddly tweaks

### PR DESCRIPTION
**Pull request by yrmvgh**
Begin to set up "penniless" weapon category for Ascetic role.
Make all backgrounds grant a more similar amount of total skill points - around 900 total. (Monks gets a free pass on gheir high skills for their lack of items)
Give fighter a javelin so he can feel like a cool dude.
Don't encourage newbies to use wand of random effects.
etc

### CODE REVIEW

File: crawl-ref/source/job-data.h [hunk 1]
```diff
@@ -60,18 +61,18 @@ static const map<job_type, job_def> job_data =
     { SP_DEEP_DWARF, SP_HALFLING, SP_KOBOLD, SP_SPRIGGAN, SP_BASE_DRACONIAN,
       SP_DEMONSPAWN, },
     { "short sword", "leather armour", "wand of flame charges:15",
-      "wand of enslavement charges:15", "wand of random effects charges:15" },
+      "wand of enslavement charges:15", "wand of frost:15" },
```
Comment (jeremygurr) [inline crawl-ref/source/job-data.h]: wands of frost don't exist

Comment (yrmvgh) [inline crawl-ref/source/job-data.h]: This is what I get for playing .17 sometimes, thank you.

**Comment by jeremygurr** [comment]
I'm not sure about most of these. I don't think the jobs need a lot more than they already have. It's a lot less cool to find some javelins in the game if you already start with some. They are pretty much the most powerful normal throwing weapon, so a player should have to work a little for them at least.

**Comment by yrmvgh** [comment]
It's just one javelin. _weeps inconsolably_

**Comment by yrmvgh** [comment]
Feel free to pick it apart and only keep the stuff you approve of. For the most part I meant to just add skill points to the roles that start with less than 900 total skill points, and remove some from roles that have over 1000. But I got a little carried away with the Editor's Sharpie!

**Comment by yrmvgh** [comment]
After:

```
Warriors:
  Fi: 954
  Gl: 904
  Mo: 1230 (!!!)
  Hu: 846
  As: 925

Adventurers:
  Ar: 897
  Wn: varies (between 593 and 947)

Zealots:
  Be: 904
  AK: 881
  CK: 904

Warrior-Mages:
  Sk: 884
  Tm: 933
  Wr: 935
  AM: 914
  En: 935

Mages: 
  Wz: 888
  Cj: 958
  Su: 958
  Ne: 958
  FE: 909
  IE: 909
  AE: 909
  EE: 909
  VM: 958
```

**Comment by yrmvgh** [comment]
Before:

```
Warriors:
  Fi: 1054
  Gl: 904
  Mo: 1230 (!!!)
  Hu: 846
  As: 925

Adventurers:
  Ar: 597 (!)
  Wn: varies (between 593 and 947)

Zealots:
  Be: 904
  AK: 781
  CK: 804

Warrior-Mages:
  Sk: 884
  Tm: 833
  Wr: 1035
  AM: 714
  En: 935

Mages: 
  Wz: 788
  Cj: 958 
  Su: 958 
  Ne: 958 
  FE: 809 
  IE: 809 
  AE: 809 
  EE: 809 
  VM: 958 
```

**Comment by jeremygurr** [comment]
I guess if you are only balancing out the skill points it makes sense. And 1 javelin is fine.

**Pull request merged**